app/ui-react/packages/ui/src/Dashboard/Dashboard.css

```diff
@@ -3,19 +3,54 @@
   padding: 0.3em 0.8em;
 }
 
+.dashboard__integrations .pf-c-card__header {
+  border-bottom: 1px solid #d1d1d1;
+}
+
 .dashboard__connections__actions {
   padding: 0 10px;
 }
 
-.dashboard__metrics .card-pf.card-pf-aggregate-status {
-  padding: 0 10px 24px 10px;
+.dashboard__metrics .pf-l-grid__item {
+  margin-bottom: var(--pf-global--spacer--md);
```

Comment:
this is from the missing var mentioned in https://github.com/syndesisio/syndesis/pull/6287/files#r308968199. If you pull in that var, you shouldn't need this.